dashboard/charts.py

```diff
@@ -0,0 +1,35 @@
+"""charts used in dashboard"""
+
+import altair as alt
+import streamlit as st
+from pandas import DataFrame
+
+
+@st.cache_data(ttl="1hr")
```

Comment:
Cacheing charts seems useful, but why are they being cached for one hour?

Reply:
Actually that's a good point. As the data taken in the arguments are being cached for an hour I can remove the time limit here as it will automatically refresh when the arguments change.